frontend/src/utils/api.js

```diff
@@ -32,6 +32,21 @@ function createEvent(event) {
   }).then(res => res.json())
 }
 
+function eventCheckin(attendee, id: string) {
+  return fetch(`${API_URL}/events/${id}/attendees?key=${getKey()}`, {
+    body: JSON.stringify({
+      name: attendee.name,
+      email: attendee.email,
+      year: attendee.year
+    }),
+    headers: {
+      'content-type': 'application/json'
+    },
+    method: 'PUT',
+    mode: 'cors'
+  }).then(res => res.json())
+}
+
```

Comment:
oops sorry didn't catch this before but you have to export the api call at the bottom of the file, that should fix your error